tools/proteore_filter_keywords_values/.shed.yml

```diff
@@ -0,0 +1,12 @@
+categories: [Proteomics]
```

Comment:
Can we have a proteore tool suite? And add all proteore tools in a single directory?

Reply:
Maybe for a few that makes sense, but they are quite diverse. So mixing python/R and such. I think suites only make sense if the tools have a lot in common.